website/other/bezier-rs-demos/wasm/src/bezier.rs

```diff
@@ -136,75 +138,219 @@ impl WasmBezier {
 			HANDLE_ATTRIBUTES.to_string(),
 			HANDLE_LINE_ATTRIBUTES.to_string(),
 		);
-		wrap_svg_tag(bezier)
+		bezier
 	}
 
-	pub fn length(&self) -> f64 {
-		self.0.length(None)
+	pub fn to_svg(&self) -> String {
+		wrap_svg_tag(self.get_bezier_path())
+	}
+
+	pub fn length(&self) -> String {
+		let bezier = self.get_bezier_path();
+		wrap_svg_tag(format!("{bezier}{}", draw_text(format!("Length: {:.2}", self.0.length(None)), 5., 193., BLACK)))
```

Comment:
Would it be beneficial to assign these constants to a `const` variable to add extra context to what they mean/why they're chosen?